### ISSUE TITLE
Recent changes in archive format made it's paths absolute

### ISSUE THREAD

**Issue by svetlyak40wt** [highlightjs/highlight.js#3835]
I noted that [download page](https://highlightjs.org/download) has changed recently and my CI process for build docs now is broken because new zip archive contains absolute pathnames:

```
% /usr/bin/unzip -l /tmp/theme/highlight-archive.zip | head
Archive:  /tmp/theme/highlight-archive.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
    38854  2023-08-04 21:16   /DIGESTS.md
     1514  2023-08-04 21:16   /LICENSE
     1717  2023-08-04 21:16   /README.md
        0  2023-08-04 21:16   /es/
    76121  2023-08-04 21:16   /es/core.js
    20322  2023-08-04 21:16   /es/core.min.js
    76121  2023-08-04 21:16   /es/highlight.js
```

Because of that, unzip (at least on Ubuntu) returns 1 exit code:

```
% /usr/bin/unzip -o /tmp/theme/highlight-archive.zip -d unpacked
...
warning:  stripped absolute path spec from /languages/markdown.min.js
 extracting: unpacked/languages/markdown.min.js
warning:  stripped absolute path spec from /languages/xml.min.js
 extracting: unpacked/languages/xml.min.js
warning:  stripped absolute path spec from /languages/yaml.min.js
 extracting: unpacked/languages/yaml.min.js

% echo $?
1
```

**Expected behavior**

Archive contains relative filenames and can be unpacked without warnings and with exit code 0.

**Comment by joshgoebel** [comment]
New website behavior?

**Comment by allejo** [comment]
Fixed in [23da75b](https://github.com/highlightjs/highlightjs.org/commit/23da75b8b5e67c531bbf93ab96a8422f9f31ba7d). Thank you for the report!
